core/src/main/java/nu/yona/server/device/service/DeviceService.java

```diff
@@ -99,15 +128,15 @@ private String getDeviceChangeMessageText(DeviceChange change, Optional<String>
 
 	public UserDeviceDto createDefaultUserDeviceDto()
 	{
-		return new UserDeviceDto(translator.getLocalizedMessage("default.device.name"), OperatingSystem.UNKNOWN);
+		return new UserDeviceDto(translator.getLocalizedMessage("default.device.name"), OperatingSystem.UNKNOWN, "Unknown");
```

Comment:
Last suggestion: please make this `"Unknown"` a const.

Reply:
I normally only do that when I use it more than once (which is not the case here).